cmake-build.el

```diff
@@ -572,6 +572,12 @@ use Projectile to determine the root on a buffer-local basis, instead.")
     (cmake-build--compile buffer-name "cmake ."
                           :other-buffer-name other-buffer-name)))
 
+(defun cmake-build--create-compile-commands-symlink ()
+  (let ((filename (expand-file-name "compile_commands.json" (cmake-build--project-root))))
```

Comment:
I'd consider creating relative not absolute symlink, it works the same for this use case but might make moving directory trees around easier for the user

Reply:
I'm not sure how to do that, any hints welcome.
Moreover if you need to recreate the link, just call configure, which is generally (more or less) fast.